e2e/rbd.go

```diff
@@ -331,7 +338,13 @@ var _ = Describe("RBD", func() {
 				if err != nil {
 					e2elog.Failf("failed to delete storageclass with error %v", err)
 				}
-				err = createRBDStorageClass(f.ClientSet, f, defaultSCName, nil, map[string]string{"csi.storage.k8s.io/fstype": "ext4"}, deletePolicy)
+				err = createRBDStorageClass(
+					f.ClientSet,
+					f,
+					defaultSCName,
+					nil,
+					map[string]string{"csi.storage.k8s.io/fstype": "ext4"},
+					deletePolicy)
```

Comment:
Since we are planning to split the parameters into different lines, It becomes difficult to read as it is harder to find the end of the function call. 
Suggested change(here and everywhere else):
In place of closing the function call with 
`deletePolicy)`
Closing it out in a separate line will make it more readable.
```
deletePolicy
)
```


E.g. 
```
err = createRBDStorageClass(
					f.ClientSet,
					f,
					defaultSCName,
					nil,
					map[string]string{"csi.storage.k8s.io/fstype": "ext4"},
					deletePolicy
                            )
```
Looks more clear to me than:
```
err = createRBDStorageClass(
					f.ClientSet,
					f,
					defaultSCName,
					nil,
					map[string]string{"csi.storage.k8s.io/fstype": "ext4"},
					deletePolicy)
```

Thoughts?

Reply:
Yes, when placing all arguments for a function on separate lines, sufficient indenting (automated by `go fmt`) and the closing `)` on a separate line too makes sense to me.